back-end/database/migrations/1741736938936-migrations.js

```diff
@@ -1,13 +1,13 @@
-const { MigrationInterface, QueryRunner } = require("typeorm");
+// const { MigrationInterface, QueryRunner } = require("typeorm");
 
-module.exports = class Migrations1741736938936 {
-    name = 'Migrations1741736938936'
+// module.exports = class Migrations1741736938936 {
+//     name = 'Migrations1741736938936'
```

Comment:
Was this manually commented out? Aren't these migration files auto generated? We shouldn't modify auto generated files as a general best practice to save you headaches. Make modifications and regenerate.